### ISSUE TITLE
LSTMP and DNN comparison

### ISSUE THREAD

**Issue by abbasov** [Microsoft/CNTK#473]
Hi,

I tested LSTMP and DNN models on 20 hours data . In contrast to my expectations DNN model showed better results than LSTMP.

Experiment results (WER):

DNN         35.72%  
LSTMP     39.37% 

Here are my configs are the same as [AN4 example configs ](https://github.com/Microsoft/CNTK/tree/master/Examples/Speech/AN4/Config)
with the following exceptions:
1. DNN with 5 layers, each has size of 2048
   `layerSizes = 1320:2048*5:6281`
2. dropout nodes are added
   `dropoutRate = 0.2`
3. LSTMP takes account 5 future frames
   
   ```
   features = [
           dim = 1320
           type = "real"
           scpFile = "$DataDir$/train.scp"
       ]
   
   basefeatDim = 120
   rowSliceStart = 1200 
   featuresSlice = RowSlice(rowSliceStart, basefeatDim, features);
   ```
   
   I wonder why such difference.
   Error rates were normal during the training:

DNN     48.6%  
LSTMP   25.78%

**Comment by dongyu888** [comment]
When you talk about training error, you are referring to frame error rate, right? frame error rates are not directly comparable between different types of models. This is because, for example in the LSTM case, history information isused when predicting the senone of current frame while this info is not used in the DNN case. So typically you will need to see much bigger gain in the frame error rate to see WER improvement which involves LM of which part of the information has been used (and learned) in LSTM.

Another issue is that due to the history info used, LSTM models are easier to overfit to the training set. So you usually see larger gains when you have use more training data.

The last issue is the learning rate. People have observed that the final model usually can perform better if they use a relatively large learning rate without diverging.

**Comment by mahilleb-msft** [comment]
Closing as being answered. Please re-open or file a new issue if necessary.
